### PR TITLE
Add UP token

### PR DESCRIPTION
**Pull request by ccarfi**
**Description**

Request to add the UP token to the Superchain token list.

Contains logo.svg and the data.json file for the UP token on Base.

### CODE REVIEW

**Comment by ccarfi** [comment]
@clemsos can you please take a quick look to review the data.json in the context of the requirements here? Thank you! https://github.com/ethereum-optimism/ethereum-optimism.github.io?tab=readme-ov-file#adding-a-token-to-the-list

**Comment by clemsos** [comment]
> @clemsos can you please take a quick look to review the data.json in the context of the requirements here? Thank you! https://github.com/ethereum-optimism/ethereum-optimism.github.io?tab=readme-ov-file#adding-a-token-to-the-list

lgtm!

**Comment by ccarfi** [comment]
@wbnns it looks like you're next up on this one. appreciate your help. thank you!

File: data/UP/data.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Unlock Protocol Token",
```
Comment (ccarfi) [inline data/UP/data.json:2]:
```suggestion
  "name": "UnlockProtocolToken",
```

File: data/UP/data.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "name": "UnlockProtocolToken",
+  "symbol": "UP",
+  "decimals": 18,
+  "description": "The Unlock Protocol UP token (UP) is the governance token of the Unlock Protocol, used for rewarding developers and creators, as well as for participating in governance through voting on proposals. It can be acquired by swapping UDT, purchasing it on decentralized exchanges, or earning it via protocol rewards and grants.",
+  "website": "https://unlock-protocol.com",
+  "twitter": "@unlockprotocol",
+  "tokens": {
+     "base": {
+      "address": "0xac27fa800955849d6d17cc8952ba9dd6eaa66187"
+    }
```
Comment (ccarfi) [inline data/UP/data.json:11]:
```suggestion
    "overrides": {
        "bridge": {
          "ethereum": "0x3154Cf16ccdb4C6d922629664174b904d80F2C35"
        }
      }
    }
```

**Comment by ccarfi** [comment]
@wbnns corrected typo in token name (removed spaces) and added bridge info (hopefully correctly), should be able to re-re-run the validate workflow. thank you!

**Review by ccarfi — COMMENTED** [review]
added nobrdige

**Review by wbnns — APPROVED** [review]
ACK

Thanks for building on Base ! 🔵

**Pull request merged**